lib/instrumentation/fastify.js

```diff
@@ -94,7 +94,7 @@ const instrumentFastify = function(fastify, opts = {}) {
         [schema.TRACE_SPAN_NAME]: "request",
         [schema.TRACE_ID_SOURCE]: spanContext.source,
         "request.host": request.hostname,
-        "request.original_url": request.req.originalUrl,
+        "request.original_url": request.raw.url,
```

Comment:
do we need to fallback here for older versions too?

```suggestion
        "request.original_url": request.raw ? request.raw.url : request.req.originalUrl,
```

maybe?

Reply:
It worked when I ran the tests with fastify v2 and with v3 in this form, but it could potentially be safer to have this  additional fallback